<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSA</class>
 <widget class="QMainWindow" name="RSA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>478</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>478</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RSA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #323450;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>458</y>
      <width>750</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 80, 54);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="consolLable">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>0</y>
       <width>450</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0;
color: rgb(52, 52, 52);</string>
     </property>
     <property name="text">
      <string>RSA</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="val_p">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>p = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inp_val_p">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>59</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(36, 36, 36);
background: #fff;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inp_val_q">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(36, 36, 36);
background: #fff;</string>
    </property>
   </widget>
   <widget class="QLabel" name="val_q">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>100</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>q = </string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="openText">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>40</y>
      <width>241</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 34, 53);
color: rgb(153, 153, 153);</string>
    </property>
   </widget>
   <widget class="QLabel" name="parametors_t">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>16</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);</string>
    </property>
    <property name="text">
     <string>Параметры</string>
    </property>
   </widget>
   <widget class="QLabel" name="open_t">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>16</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);</string>
    </property>
    <property name="text">
     <string>Открытый Текст</string>
    </property>
   </widget>
   <widget class="QLabel" name="val_fn">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>150</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>Ф(n) = </string>
    </property>
   </widget>
   <widget class="QLabel" name="val_fn_number">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pnq_button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="val_n">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>180</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>n = </string>
    </property>
   </widget>
   <widget class="QLabel" name="val_n_number">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="val_e">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>240</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>e = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inp_val_e">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>238</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(36, 36, 36);
background: #fff;</string>
    </property>
   </widget>
   <widget class="QLabel" name="np_e">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>нп.</string>
    </property>
   </widget>
   <widget class="QLabel" name="np_e_number">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="np_k">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>k =</string>
    </property>
   </widget>
   <widget class="QLabel" name="np_k_number">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="np_d">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>d =</string>
    </property>
   </widget>
   <widget class="QLabel" name="np_d_number">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="clousText">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>241</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 34, 53);
color: rgb(153, 153, 153);</string>
    </property>
   </widget>
   <widget class="QLabel" name="clous_t">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>16</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);</string>
    </property>
    <property name="text">
     <string>Закрытый Текст</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shifrButton">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>380</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>Шифровать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deshifrButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
background-color: rgb(33, 34, 53);</string>
    </property>
    <property name="text">
     <string>Дешифровать</string>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>val_p</zorder>
   <zorder>inp_val_q</zorder>
   <zorder>val_q</zorder>
   <zorder>openText</zorder>
   <zorder>parametors_t</zorder>
   <zorder>open_t</zorder>
   <zorder>val_fn</zorder>
   <zorder>val_fn_number</zorder>
   <zorder>pnq_button</zorder>
   <zorder>inp_val_p</zorder>
   <zorder>val_n</zorder>
   <zorder>val_n_number</zorder>
   <zorder>val_e</zorder>
   <zorder>inp_val_e</zorder>
   <zorder>np_e</zorder>
   <zorder>np_e_number</zorder>
   <zorder>np_k</zorder>
   <zorder>np_k_number</zorder>
   <zorder>np_d</zorder>
   <zorder>np_d_number</zorder>
   <zorder>clousText</zorder>
   <zorder>clous_t</zorder>
   <zorder>shifrButton</zorder>
   <zorder>deshifrButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
